Server/Server.cpp

```diff
@@ -52,10 +56,10 @@ unsigned short Server::_getAFreePort(unsigned short actual)
     }
 }
 
-unsigned short Server::createANewRoom(std::string name)
+unsigned short Server::createANewRoom(std::string name, short *configs)
```

Comment:
👀

Reply:
configurations are used to add modifier on room creation. There are raw pointer because they are used by the network.